docs/content/topics/operators/security.md

```diff
@@ -19,28 +19,35 @@ For that reason, it is prudent to configure security.
   in a Helm install, do `helm install --namespace brigade ...`.
 - *RBAC Enabled*: When installing with Helm, role-based access control is enabled by default.
   To disable,`--set rbac.enabled=false` will turn off role-based access control.
-- *Do not run more than one brigade per namespace*: Running multiple installs of Brigade
+- *Do not run more than one Brigade per namespace*: Running multiple installs of Brigade
   in the same namespace can cause naming collisions which could result in
   unauthorized access to pods.
-- *Do not run multiple tenants against the same brigade*: Brigade does not implement
-  security controls that allow multiple tenants to share the same Brigade instance.
-  Brigade supports multiple projects per Brigade server instance, but those projects
-  should be owned by the same tenant.
 
 ## API Server Security
 
-TODO: update
+If Brigade's API server will be exposed to the internet, e.g. via an external
+IP (recall that Brigade's default service type in the [Brigade chart] is
+`LoadBalancer` and so a public-facing IP will be provisioned), care should be
+taken to secure inbound communication. Minimally, TLS should be enabled and 
+SSL certificates should not be self-signed. You may also consider using an
+[Ingress Controller] for an extra layer of traffic filtering and/or protection.
+
+For more details on deploying Brigade securely, see the [Deployment] doc.
+
+[Brigade chart]: https://github.com/brigadecore/brigade/tree/v2/charts/brigade
+[Ingress Controller]: https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/
+[Deployment]: /topics/operators/deploy
 
 ## How RBAC Is Configured
 
-The Helm [chart](https://github.com/brigadecore/brigade/tree/v2/charts/brigade) for Brigade
-includes an RBAC configuration that is designed to run in an isolated namespace.
+The Helm [chart][Brigade chart] for Brigade includes an RBAC configuration that
+is designed to run in an isolated namespace.
 
-The RBAC defines a Service Account for every Deployment (API, Observer, Scheduler, etc.),
-as well as a Service Account for the Brigade worker.
+The RBAC defines a Service Account for every Deployment (API, Observer,
+Scheduler, etc.), as well as a Service Account for the Brigade worker.
```

Comment:
Each project gets its own namespace. Within each of those, there's a service account for the workers and another for the jobs.

This allows competent k8s users with adequate permissions to modify the service account(s) for a given project to permit things that you can't do otherwise-- such as scripting k8s itself.